.ci.yaml

```diff
@@ -7,47 +7,129 @@
 #  * https://github.com/flutter/cocoon/blob/master/CI_YAML.md
 enabled_branches:
   - master
+  - dev
+  - beta
+  - stable
+  - flutter-2.4-candidate.4
+
+platform_properties:
+  linux:
+    properties:
+      caches: >-
+        [
+          {"name":"builder_linux_engine","path":"builder"}
+        ]
+      os: Linux
+  mac:
+    properties:
+      caches: >-
+        [
+          {"name":"flutter_cocoapods","path":"cocoapods"},
+          {"name":"osx_sdk","path":"old_osx_sdk"},
+          {"name":"builder_mac_engine","path":"builder"}
+        ]
+      os: Mac-10.15
+      xcode: "11e708" # Xcode 12
```

Comment:
Why does this need to explicitly be a string? Shouldn't it be loaded as a json string anyways?

Reply:
I had a pre-submit ci.yaml validation error. I think a value that starts with an integer is assumed to be a number, and then the `e` made it a parse error? Using this https://www.convertjson.com/yaml-to-json.htm I get `11e708` -> `null`

Reply:
I'll give github credit, they have a yaml syntax coloring in their diff view, and the `11e708` was a different color than the strings.

Reply:
ok, it's actually `11e708` is parsed as a number in scientific notation.